migrations/Version202209141533261465_qtiItemPci.php

```diff
@@ -34,7 +33,7 @@ public function up(Schema $schema): void
     public function down(Schema $schema): void
     {
         throw new IrreversibleMigration(
-            'In order to undo this migration, please revert the client-side changes and run ' . RegisterPciMathEntry::class
+            'To undo this migration, please revert the client-side changes and run ' . RegisterPciMathEntry::class
```

Comment:
if you change this the translation will break as we are using the string inside as keys